client/login/wp-login/index.jsx

```diff
@@ -79,6 +79,9 @@ export class Login extends React.Component {
 
 					<DocumentHead title={ translate( 'Log In', { textOnly: true } ) } />
 
+					<DocumentHead meta={ [ { name: 'robots', content: 'noindex, follow' } ] } />
+					<DocumentHead link={ [ { rel: 'canonical', href: 'https://wordpress.com/wp-login.php' } ] } />
```

Comment:
What do you think of using our [`login()` path function](https://github.com/Automattic/wp-calypso/blob/master/client/lib/paths/login/index.js) here instead of hardcoding the url?